worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -42,24 +42,27 @@ import org.apache.celeborn.service.deploy.worker.storage.{FileWriter, HdfsFlushe
 
 class PushDataHandler extends BaseMessageHandler with Logging {
 
-  var workerSource: WorkerSource = _
-  var partitionLocationInfo: WorkerPartitionLocationInfo = _
-  var shuffleMapperAttempts: ConcurrentHashMap[String, AtomicIntegerArray] = _
-  var shufflePartitionType: ConcurrentHashMap[String, PartitionType] = _
-  var shufflePushDataTimeout: ConcurrentHashMap[String, Long] = _
-  var replicateThreadPool: ThreadPoolExecutor = _
-  var unavailablePeers: ConcurrentHashMap[WorkerInfo, Long] = _
-  var pushClientFactory: TransportClientFactory = _
-  var registered: AtomicBoolean = new AtomicBoolean(false)
-  var workerInfo: WorkerInfo = _
-  var diskReserveSize: Long = _
-  var partitionSplitMinimumSize: Long = _
-  var shutdown: AtomicBoolean = _
-  var storageManager: StorageManager = _
-  var conf: CelebornConf = _
-  @volatile var pushMasterDataTimeoutTested = false
-  @volatile var pushSlaveDataTimeoutTested = false
-  var workerPartitionSplitEnabled: Boolean = _
+  private var workerSource: WorkerSource = _
+  private var partitionLocationInfo: WorkerPartitionLocationInfo = _
+  private var shuffleMapperAttempts: ConcurrentHashMap[String, AtomicIntegerArray] = _
+  private var shufflePartitionType: ConcurrentHashMap[String, PartitionType] = _
+  private var shufflePushDataTimeout: ConcurrentHashMap[String, Long] = _
+  private var replicateThreadPool: ThreadPoolExecutor = _
+  private var unavailablePeers: ConcurrentHashMap[WorkerInfo, Long] = _
+  private var pushClientFactory: TransportClientFactory = _
+  private var registered: AtomicBoolean = new AtomicBoolean(false)
```

Comment:
could be `val`?

Reply:
No, the variable `shutdown` is assigned as a reference to `worker.shutdown` during the class initialization process.

Reply:
the initial value could be eliminated then?

Reply:
updated.